src/authorization/ExtensionAclManager.ts

```diff
@@ -15,7 +15,7 @@ import { AclManager } from './AclManager';
  * It is potentially possible that an agent wants to generate the acl file before generating the actual file.
  * (Unless this is not allowed by the spec, need to verify).
  */
-export class SimpleExtensionAclManager implements AclManager {
+export class ExtensionAclManager implements AclManager {
```

Comment:
`ExtensionBasedAclManager`? And maybe have the extension as constructor param at some point.

Reply:
Or `UrlBasedAclManager` even.